### ISSUE TITLE
The weapon names dont translate in any language (1.19.2)

### ISSUE THREAD

**Issue by alexvirgo13** [ObliviousSpartan/SpartanWeaponry-API#33]
I don't know if its because of the way the weapons get their names since they are made up of a material and a weapon type but they dont change when changing the language to any of the supported languages.

There's also the same problem for newer items like the simple pole, simple handle and some of the oils (Not all of them, some work and some are half translated).

The mob heads' translations seem to work just fine.

**Comment by ObliviousSpartan** [comment]
I just checked the translation file you gave me a while back and I should've noticed the problem a lot earlier.
It is because when I was developing version 3.x of Spartan Weaponry (1.18.2, 1.19.x and 1.20.x and possibly above), I changed the registry names for all of the weapons to more closely match the vanilla naming convention.
For example, 
"spartanweaponry:dagger_wood" became "spartanweaponry:wooden_dagger",
"spartanweaponry:greatsword_iron" became "spartanweaponry:iron_greatsword"
etc.
Note that some material names were changed; "wood" to "wooden" and "gold" to "golden"

The correct names are in the English version of the file here:
https://github.com/ObliviousSpartan/SpartanWeaponry/blob/badb5ae11853b56aec6f4be73512c6b9d9d92505/src/main/resources/assets/spartanweaponry/lang/en_us.json

This should be relatively easy to fix. I should also look through the other translations to ensure they work.
